WNPRC_EHR/src/org/labkey/wnprc_ehr/notification/IrregularObsBehaviorNotification.java

```diff
@@ -176,7 +179,9 @@ private StringBuilder getBehavioralIrregularObs (Container c, User u, LocalDateT
                 msg.append("</table>");
             }
         } catch (SQLException e) {
-            return new StringBuilder("<strong>There was an error retrieving the data. Please alert the IDS team.</strong>");
+            System.err.println("Error retrieving IrregularObs data: " + e.getMessage());
+            e.printStackTrace(System.err);
```

Comment:
Not required for merging, but strongly recommend using a Logger so output goes to labkey.log instead of std err